--- a/config/sl_uartdrv_usart_vcom_config.h	2023-04-12 09:15:26.328472517 +0200
+++ b/config/sl_uartdrv_usart_vcom_config.h	2023-04-12 09:19:41.174293789 +0200
@@ -75,17 +75,17 @@
 // <<< sl:start pin_tool >>>
 // <usart signal=TX,RX,(CTS),(RTS)> SL_UARTDRV_USART_VCOM
 // $[USART_SL_UARTDRV_USART_VCOM]
-#warning "UARTDRV USART peripheral not configured"
-// #define SL_UARTDRV_USART_VCOM_PERIPHERAL        USART0
-// #define SL_UARTDRV_USART_VCOM_PERIPHERAL_NO     0
-
-// #define SL_UARTDRV_USART_VCOM_TX_PORT           gpioPortA
-// #define SL_UARTDRV_USART_VCOM_TX_PIN            0
-// #define SL_UARTDRV_USART_VCOM_TX_LOC            0
-
-// #define SL_UARTDRV_USART_VCOM_RX_PORT           gpioPortA
-// #define SL_UARTDRV_USART_VCOM_RX_PIN            1
-// #define SL_UARTDRV_USART_VCOM_RX_LOC            0
+// #warning "UARTDRV USART peripheral not configured"
+#define SL_UARTDRV_USART_VCOM_PERIPHERAL        USART0
+#define SL_UARTDRV_USART_VCOM_PERIPHERAL_NO     0
+
+#define SL_UARTDRV_USART_VCOM_TX_PORT           gpioPortA
+#define SL_UARTDRV_USART_VCOM_TX_PIN            0
+#define SL_UARTDRV_USART_VCOM_TX_LOC            0
+
+#define SL_UARTDRV_USART_VCOM_RX_PORT           gpioPortA
+#define SL_UARTDRV_USART_VCOM_RX_PIN            1
+#define SL_UARTDRV_USART_VCOM_RX_LOC            0
 
 // #define SL_UARTDRV_USART_VCOM_CTS_PORT          gpioPortA
 // #define SL_UARTDRV_USART_VCOM_CTS_PIN           2
